resources/functionMap.php

```diff
@@ -11967,7 +11967,9 @@
 'stomp_version' => ['string'],
 'StompException::getDetails' => ['string'],
 'StompFrame::__construct' => ['void', 'command='=>'string', 'headers='=>'array', 'body='=>'string'],
+'str_decrement' => ['non-empty-string'],
 'str_getcsv' => ['non-empty-list<?string>', 'input'=>'string', 'delimiter='=>'string', 'enclosure='=>'string', 'escape='=>'string'],
+'str_increment' => ['non-falsy-string'],
```

Comment:
Since `str_increment(-1)` is not accepted and never returns either `""` or `"0"`, it is typed as `non-falsy-string`. https://3v4l.org/59U2u#v8.3.0